### ISSUE TITLE
With regards to ressource usage

### ISSUE THREAD

**Issue by yan-foto** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1111]
Hi there! This is not an issue to report any bugs or ask for new features.

I noticed that you advertise "resource efficiency" on the project's web page comparing to running Google Play Music in a browser. Considering that Electron is nothing but Chromium + Node.js on steroids and you use a `webview` tag to embed Google's web player, how do you manage to improve resource efficiency?

**Comment by MarshallOfSound** [comment]
1.  Chrome as a whole uses more resources than Chromium (important distinction)
2.  Running a single website in an Electron webview is (by nature) more efficient than running an entire browser
3.  In Chrome / FF you would need a multitude of extensions to get the features this player gives you.  In chrome each extension runs as a separate process which is horrific for resources consumption.

TLDR: By nature of implementation this player is more efficient :+1:
